Player.py

```diff
@@ -107,20 +116,26 @@ def steal_mana(self, mana_to_steal):
 		selected_organism.change_num_mana(-mana_to_steal)
 		return mana_to_steal
 
-	def give_extra_move(self, now):
+	def give_extra_move(self, now, move_bonus = 1, move_bonus_duration = 1):
```

Comment:
Yeah as mentioned before, just change this to take no args and give an extra move if possible.